Modules/_decimal/_decimal.c

```diff
@@ -1522,6 +1552,7 @@ static PyObject *
 current_context_from_dict(void)
 {
     PyThreadState *tstate = _PyThreadState_GET();
+    decimal_state *state = GLOBAL_STATE();
```

Comment:
I'd use `modstate` or another more explicit name for this particular scope; the difference between `tstate` and `state` is small; it is easy to write or interpret it wrong. Explicit is nice.